dbt_platform_helper/utils/aws.py

```diff
@@ -18,6 +18,9 @@
 SSM_BASE_PATH = "/copilot/{app}/{env}/secrets/"
 SSM_PATH = "/copilot/{app}/{env}/secrets/{name}"
 AWS_SESSION_CACHE = {}
+REFRESH_TOKEN_MESSAGE = (
+    "To refresh this SSO session run `aws sso login` with the corresponding profile"
+)
```

Comment:
I would be inclined to declare this in `get_aws_session_or_abort` so it's scoped to that method.

The tests can remain as strings, not using this, otherwise you could splurge a typo into here and the test would still pass.

Reply:
I rescoped the var, was this what you meant RE strings in tests?

Reply:
The test could have its own variable to avoid duplication, but it should not be the one in the production code. Up to you whether you do that last little optional refactor. I am approving.